pkgs/by-name/hy/hyprland/package.nix

```diff
@@ -45,30 +46,50 @@
   enableNvidiaPatches ? false,
 }:
 let
-  info = builtins.fromJSON (builtins.readFile ./info.json);
+  inherit (builtins)
+    foldl'
+    fromJSON
+    readFile
+    ;
+  inherit (lib.asserts) assertMsg;
+  inherit (lib.attrsets) mapAttrsToList;
+  inherit (lib.lists)
+    flatten
+    concatLists
+    optionals
+    ;
+  inherit (lib.strings)
+    makeBinPath
+    optionalString
+    mesonBool
+    mesonEnable
+    ;
```

Comment:
words can hardly express how much I hate this

Reply:
The art of the glorious nixfmt.